frontend/src/components/common/ProductDisplay/ProductList/Product/Product.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Product.scss';
+
+interface IProp {
+  isLoading: boolean;
+  id: number;
+  name: string;
+  price: number;
+  imageUrl: string;
+}
+
+function Product(props: IProp) {
+  const { isLoading, name, price, imageUrl } = props;
+  console.log("loading: ", isLoading);
```

Comment:
don't include console.log or debug or unnecessary comments in the code when creating PR